dps-gradle-spring-boot-suppressions.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can't really remember - but don't think this file needs to be checked in?  IIRC the trivy one does but not this one...